### PR TITLE
Add new option to setting 'emailconfirmation'

### PR DESCRIPTION
**Pull request by gleimermora**
#37

### CODE REVIEW

File: version.php [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
 defined('MOODLE_INTERNAL') || die;
 
-$plugin->version   = 2020030200;        // The current plugin version (Date: YYYYMMDDXX).
+$plugin->version   = 2020030300;        // The current plugin version (Date: YYYYMMDDXX).
 $plugin->release   = 2020030200;        // Match release exactly to version.
```
Comment (brendanheywood) [inline version.php]: please keep the release in lock step

Comment (gleimermora) [reply version.php]: updated

File: auth.php [hunk 2]
```diff
@@ -116,7 +116,21 @@ public function user_signup($user, $notify=true) {
         $user->picture = 0;
         $user->imagealt = 0;
         $user->deleted = 0;
-        $user->policyagreed = (int)!get_config('auth_enrolkey', 'emailconfirmation');
+        $setconfirmationlevel = function () use (&$user) {
```
Comment (brendanheywood) [inline auth.php]: this seems weird to me

Comment (gleimermora) [reply auth.php]: done

**Pull request merged**